kolena/workflow/evaluator_function.py

```diff
@@ -246,3 +246,31 @@ def _test_sample_key(ts: TestSample) -> str:
 def _is_configured(evaluator: BasicEvaluatorFunction) -> bool:
     param_values = list(signature(evaluator).parameters.values())
     return len(param_values) == 5 and issubclass(param_values[4].annotation, EvaluatorConfiguration)
+
+
+def noop_evaluator(
```

Comment:
Slight preference here for `no_op` over `noop`. What do you think?

Reply:
no strong preference, happy to rename